### DISCUSSION TITLE
Node js upgrade warning [Node.js 12 actions are deprecated. Please update the following actions to use Node.js 16]

### DISCUSSION THREAD

**Discussion by vasudevan-n** [community/community#48035] [Actions]
### Select Topic Area

Product Feedback

### Body

if we dont use javacript in our code ,we use only powershell do we still need to upgrade the node js version ? Is Node.js internally used by Github to run actions ? We get the above error message we see in Subject of this conversation. I am new to github pipelines  and any suggestions really help

**Comment by vitalykarasik** [comment]
This warning is regarding Node.js version which uses GitHub actions in your pipeline. for example, 'checkout', 'configure-aws-credentials', etc.
So you should review all Giuhub actions you use, and upgrade their version. Unfortunately, many 3rd party action haven't updated nodejs yet.
